x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in high mean of transaction duration (ECS).",
+  "description": "Detect anomalies in transactions from your APM services",
   "type": "Transaction data",
   "logoFile": "logo.json",
-  "defaultIndexPattern": "apm-*",
+  "defaultIndexPattern": "apm-*-transaction",
   "query": {
     "bool": {
       "filter": [
         { "term": { "processor.event": "transaction" } },
-        { "term": { "transaction.type": "request" } }
```

Comment:
Was the previous job tied to `transaction.type=request`? 🤔

Reply:
That change happened in https://github.com/elastic/kibana/pull/30820 - does that mean ML did not work for other transaction types for the past 1.5 years?

Reply:
yes, if the jobs were created via the ML job creation process, then the datafeed filtered for `"transaction.type": "request"`.